packages/manager/apps/pci-instances/src/pages/instances/instance/component/InstanceGeneralDetails.component.tsx

```diff
@@ -88,9 +88,20 @@ const InstanceGeneralDetails: FC = () => {
       <TileBlock label={t('pci_instances_dashboard_price_title')}>
         <LoadingCell isLoading={isLoading}>
           <div className="flex flex-col">
-            {instance.prices.map(({ value, type }) => (
-              <PriceLabel key={type} value={value} type={type} />
-            ))}
+            <div>
+              {instance.prices.map(({ value, type, label }) => (
+                <div className="my-4" key={type}>
```

Comment:
maybe you can change the key value here, as it does not seem to be unique, or am I wrong?